superset/queries/saved_queries/api.py

```diff
@@ -252,3 +265,135 @@ def export(self, **kwargs: Any) -> Response:
             as_attachment=True,
             attachment_filename=filename,
         )
+    @expose("/import/", methods=["POST"])
```

Comment:
You also need to add `RouteMethod.IMPORT` to `include_route_methods` in this file, otherwise the endpoint will not be exposed.

Reply:
oh, I hadn't noticed that. Added.